tsl/test/src/test_ddl_hook.c

```diff
@@ -187,6 +187,10 @@ Datum
 ts_test_ddl_command_hook_reg(PG_FUNCTION_ARGS)
 {
 	Assert(ts_cm_functions->ddl_command_start == NULL);
+
+	/* remove me */
+	abort();
```

Comment:
For the testing purposes to see it in action

Reply:
I suggest to post link to the build with the failures due to this line and revert this line.